docs/draft/6-payloads.md

```diff
@@ -319,6 +342,80 @@ A client SHOULD send a `ContactUpdate` to all the contacts each time:
 
 A client SHOULD also periodically send a `ContactUpdate` to all the contacts, the interval is up to the client, the Status official client sends these updates every 48 hours.
 
+### Contact
+
+`Contact` is a representation of other accounts encountered on the app. They are added through `ContactUpdate`s propagated.
+A `Contact` does not mean a "friend". Any account from which we receive a `ContactUpdate` is added to the table.
+
+Users are considered "Friends" when they are **mutual** contacts, meaning that both of them have the `added` and `requestReceived` boolean set as `true`, which then triggers setting `mutualContact` to `true`. This is done through `ContactRequest`s.
+
+#### Payload
+
+| Field | Name | Type | Description |
+| ----- | ---- | ---- | ---- |
+| 1 | id | `string` |  ID of the contact. Hex-encoded public key (prefixed with 0x). |
+| 2 | address | `string` | Ethereum address of the contact |
```

Comment:
I think this is a legacy field, we used to have it once wallet was derived by the public key, but in case pk is enough.